bower.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "o-share",
   "dependencies": {
-    "ftdomdelegate": "^3.1.0",
-    "o-colors": "^4.0.2",
-    "o-normalise": "^1.2.1",
+    "ftdomdelegate": "#breaking",
```

Comment:
Origami needs to have versions which work in bower and npm. `#breaking` does not work in npm, npm requires one of these two formats:
- `git://github.com/<user>/<project>.git#<branch>`
- `<user>/<project>#<branch>` (If using github)

Reply:
This won't be published until these are updated. My intention is to ignore the occ error until the low level components are stable -- then I'll release betas again.

Reply:
Okay I did end up going back to betas. Seeing all the failing builds was too much (and hard to tell if it was just occ or an actual issue). Hopefully I won't need to release another low level beta.